src/main/java/frc/robot/subsystems/lifter/Lifter.java

```diff
@@ -0,0 +1,69 @@
+package frc.robot.subsystems.lifter;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.utils.Conversions;
+import frc.utils.GBSubsystem;
+import org.littletonrobotics.junction.Logger;
+
+public class Lifter extends GBSubsystem {
+
+	private final ControllableMotor motor;
+	private final LifterStuff lifterStuff;
+	private final LifterCommandsBuilder lifterCommandsBuilder;
+
+	public Lifter(LifterStuff lifterStuff) {
+		super(lifterStuff.logPath());
+		this.motor = lifterStuff.motor();
+		this.lifterStuff = lifterStuff;
+		this.lifterCommandsBuilder = new LifterCommandsBuilder(this);
+
+		motor.resetPosition(new Rotation2d());
+
+		updateInputs();
+	}
+
+	public void setPower(double power) {
+		motor.setPower(power);
+	}
+
+	public void stop() {
+		motor.stop();
+	}
+
+	public void setBrake(boolean brake) {
+		motor.setBrake(brake);
+	}
+
+	public boolean isHigher(double expectedPosition) {
+		return expectedPosition < convertToMeters(lifterStuff.positionSignal().getLatestValue());
+	}
+
+	public boolean isLower(double expectedPosition) {
+		return !isHigher(expectedPosition);
```

Comment:
What if its at position? What is the use for this

Reply:
we not doin PID we doing set power and isHigher or is LOWer from targer (if extending useing isHiogher, if retracting, using isLower)

Reply:
@bossyshrimp68